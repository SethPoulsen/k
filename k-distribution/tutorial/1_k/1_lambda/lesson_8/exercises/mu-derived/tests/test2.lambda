
letrec times2 x = if x <= 1 then 2 else (2 + (times2 (x + -1)))
in (times2 100)