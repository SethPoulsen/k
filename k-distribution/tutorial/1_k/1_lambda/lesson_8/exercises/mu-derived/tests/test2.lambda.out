<k>
  200
</k>
